### PR TITLE
Add support of an array of price buckets and standard granuarlities

### PR DESCRIPTION
**Pull request by mike-chowla**
Update to PR #104 - Adding tests and README changes

Address Issue #27 Auto / Dense price granularity

Adds supports for an array of price buckets as well as standard Prebid price granularities: auto, dense, medium, low

### CODE REVIEW

**Comment by mike-chowla** [comment]
This PR also obsoletes #29

File: README.md [hunk 1]
```diff
@@ -69,7 +69,7 @@ Setting | Description | Type
 `DFP_TARGETED_PLACEMENT_NAMES` | The names of GAM placements the line items should target | array of strings
 `DFP_PLACEMENT_SIZES` | The creative sizes for the targeted placements | array of objects (e.g., `[{'width': '728', 'height': '90'}]`)
 `PREBID_BIDDER_CODE` | The value of [`hb_bidder`](http://prebid.org/dev-docs/publisher-api-reference.html#module_pbjs.bidderSettings) for this partner | string
-`PREBID_PRICE_BUCKETS` | The [price granularity](http://prebid.org/dev-docs/publisher-api-reference.html#module_pbjs.setPriceGranularity); used to set `hb_pb` for each line item | object
+`PREBID_PRICE_BUCKETS` | The [price granularity](https://docs.prebid.org/dev-docs/publisher-api-reference/setConfig.html#setConfig-Price-Granularity); used to set `hb_pb` for each line item.  Can use Prebid standard granularities such as `medium` or `dense`, a single custom CPM bucket object or an array of custom CPM bucket objects | object, arary or string
```
Comment (kmjennison) [inline README.md:72]: Typo: arary --> array

File: tests/test_add_new_prebid_partner.py [hunk 2]
```diff
@@ -185,6 +185,75 @@ def test_user_confirmation_accepted(self, mock_input,
     tasks.add_new_prebid_partner.main()
     mock_setup_partners.assert_called_once()
 
+  @patch('tasks.add_new_prebid_partner.setup_partner')
+  @patch('tasks.add_new_prebid_partner.input', return_value='y')
+  def test_prebid_medium_price_bucket_granularity(self, mock_input, mock_setup_partners, mock_dfp_client):
+    """
+    Sets prices buckets correctly based on prebid auto granularity
```
Comment (kmjennison) [inline tests/test_add_new_prebid_partner.py:192]: Typo: should be prebid "medium" granularity

**Review by kmjennison — CHANGES_REQUESTED** [review]
Thanks for these updates! Two typos in documentation, then LGTM.

File: README.md (hunk 1 above)
Comment (mike-chowla) [reply README.md]: Fixed

File: tests/test_add_new_prebid_partner.py (hunk 2 above)
Comment (mike-chowla) [reply tests/test_add_new_prebid_partner.py]: Fixed

**Review by kmjennison — APPROVED** [review]
Thanks for the contribution!

**Pull request merged**